base_station/gui/src/components/Amino.vue

```diff
@@ -6,37 +6,37 @@
   <div class="box1">
     <label for="site">Choose a site:</label>
     <select v-model = "site" name="site" id="site">
-      <option value="2">Site A</option>
-      <option value="1">Site B</option>
-      <option value="0">Site C</option>
+      <option value="0">Site 0</option>
+      <option value="1">Site 1</option>
+      <option value="2">Site 2</option>
     </select>
   </div>
   <div class="box1" v-if="site == 2">

```

Comment:
could probably rearrange this so that site 0 comes in first, doesn't really matter though